### ISSUE TITLE
Training Loss low but no detections

### ISSUE THREAD

**Issue by gplast** [Tianxiaomo/pytorch-YOLOv4#78]
Hello and thank you for this great repo and keep up the great work :) !

The problem that i'm facing is that, even thought the loss is below zero while training, i cannot get any detections while running my demo.

I've used your code to train another network and not the exact YOLOV4 architecture, but i've changed the anchors, strides etc. in order to make it work.

Do you have any idea what might cause this problem? I've try a series of different networks and i'm always getting the same behavior. Do you have something hard coded on the loss function or detection other than the anchors, inputs, strides, masks?

Thank you in advance.
